### PR TITLE
Development prep dockerize 🐳 📦 

### PR DESCRIPTION
**Pull request by tomveugelers**

### CODE REVIEW

File: client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM node:8.9.4-alpine
```
Comment (EHbng) [inline client/Dockerfile]: isnt this a very old version?

File: server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM node:8.9.4-alpine
```
Comment (EHbng) [inline server/Dockerfile]: same here

**Comment by tomveugelers** [comment]
small PR's are overrated...

File: blockchain/network/peerOrganizations/org1.example.com/peers/peer1.org1.example.com/peer.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      - CORE_LEDGER_STATE_COUCHDBCONFIG_COUCHDBADDRESS=couch.peer1.org1.example.com:5984
+      - CORE_LEDGER_STATE_COUCHDBCONFIG_USERNAME=
+      - CORE_LEDGER_STATE_COUCHDBCONFIG_PASSWORD=
+    volumes:
+      - ./tls:/crypto/tls
+      - ./msp:/crypto/msp
+#      - ./data/:/var/hyperledger/production
+    ports:
+      - 8051:7051
+      - 8053:7053
```
Comment (EHbng) [inline blockchain/network/peerOrganizations/org1.example.com/peers/peer1.org1.example.com/peer.yml]: new line

File: blockchain/network/shared/scripts/generate.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo "Create crypto here"
```
Comment (EHbng) [inline blockchain/network/shared/scripts/generate.sh]: new line.

File: blockchain/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "commonjs",
+    "strict": true
+  }
+}
```
Comment (EHbng) [inline blockchain/tsconfig.json]: new line

**Pull request merged**